thirdeye-ui/src/app/pages/welcome-page/create-datasource/onboard-datasets/onboard-datasets-page.component.tsx

```diff
@@ -217,6 +217,7 @@ export const WelcomeSelectDatasets: FunctionComponent = () => {
                                         <Box
                                             alignItems="flexStart"
                                             display="flex"
+                                            id="datasets-options-container"
```

Comment:
Any specific reason for using constants in some places and hardcoded strings in other?

Reply:
good point, let me convert to constants